armory/art_experimental/attacks/poisoning_attack_backdoor.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import numpy as np
-from art.attacks.attack import PoisoningAttack
+from art.attacks import PoisoningAttack
```

Comment:
I believe this will break until we update to art-1.1.1

Reply:
Are there current issues preventing updating to art-1.1.1, or do we want to wait for 1.2?